.circleci/config.yml

```diff
@@ -17,6 +17,21 @@ jobs:
       - deploy:
           name: Trigger Jenkins deploy
           command: bin/jenkins_build deploy-ci-app-catalog-next
+  
+  test_importer: 
+    docker:
+      - image: circleci/python:3.6
+    steps:
+      - checkout
+      - run:
+          name: tests 
+          command: | 
+            cd tools/harvest_source_import
+            pip install --upgrade pip 
+            pip install -r dev-requirements.txt
+            flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+            flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+            python -m pytest --vcr-record=none tests/
```

Comment:
These steps are worth adding to a local subdir makefile so it's easy to test locally as well.

Also, with the addition of Pipenv, we should use a Pipfile instead of a requirements.txt.

Reply:
Done [here](https://github.com/GSA/catalog.data.gov/commit/2450e45e54669b29cdeb29d4c5c1291983b1195c)

Reply:
Pipenv?